conda-forge.yml

```diff
@@ -1,2 +1,2 @@
-max_py_ver: '37'
-max_r_ver: '35'
+max_py_ver: '38'
+max_r_ver: '36'
```

Comment:
Please don't modify files outside of the recipe dir.

Reply:
Instead use version constraints in the recipe itself.  Though this is a general change that we probably should make

Reply:
Thanks, done in https://github.com/conda-forge/staged-recipes/pull/12093/commits/6d503160575bfaa094611e0fb957fcde31b6d6e7